### PR TITLE
[Improve] AssertUtils add and check not null improvements

### PR DESCRIPTION
**Pull request by wolfboys**
[Improve] AssertUtils add and check not null improvements

## What changes were proposed in this pull request

Issue Number: close #xxx 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @RocMarshal PTAL, thx

File: streampark-common/src/main/java/org/apache/streampark/common/util/ExceptionUtils.java
Comment (RocMarshal) [inline streampark-common/src/main/java/org/apache/streampark/common/util/ExceptionUtils.java:1]: Does this change seem unrelated to the current ISSUE title?

Perhaps it would be better to record it separately in an ISSUE.

File: streampark-common/src/main/scala/org/apache/streampark/common/util/AssertUtils.scala [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+import org.apache.streampark.common.util.Utils.isEmpty
+
+import javax.annotation.Nullable
+
+import java.util
+
+import scala.collection.convert.ImplicitConversions._
+
+/** @since 2.2.0 */
+object AssertUtils {
```
Comment (RocMarshal) [inline streampark-common/src/main/scala/org/apache/streampark/common/util/AssertUtils.scala:28]: How about 
`AssertUtils` -> `Premises` ?

**Review by RocMarshal — APPROVED** [review]
Hi, @wolfboys  Nice pr!. Thank you for your hard work.
LGTM on the whole.

Just a few of comments. PTAL in your free time~

File: streampark-common/src/main/scala/org/apache/streampark/common/util/AssertUtils.scala (hunk 1 above)
Comment (wolfboys) [reply streampark-common/src/main/scala/org/apache/streampark/common/util/AssertUtils.scala:28]: > How about `AssertUtils` -> `Premises` ?

oh, I think AssertUtils is a nice name, e.g: the name in spring is [Assert](https://github.com/spring-projects/spring-framework/blob/5.1.x/spring-core/src/main/java/org/springframework/util/Assert.java)

**Review by caicancai — APPROVED** [review]
LGTM
